src/components/dropdown-menu/index.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+export { DropdownMenu } from './DropdownMenu';
+export { DropdownMenuGroupLabel } from './DropdownMenuGroupLabel';
+export { DropdownMenuItemButtonCheckable } from './DropdownMenuItemButtonCheckable';
+export { DropdownMenuItemButtonCopy } from './DropdownMenuItemButtonCopy';
+export { DropdownMenuItemButtonDefault } from './DropdownMenuItemButtonDefault';
+export { DropdownMenuItemButtonDestructive } from './DropdownMenuItemButtonDestructive';
+export { DropdownMenuItemButtonDownload } from './DropdownMenuItemButtonDownload';
+export { DropdownMenuItemLink } from './DropdownMenuItemLink';
+export { DropdownMenuSeparator } from './DropdownMenuSeparator';
```

Comment:
@jeremy-davis-sonarsource @david-cho-lerat-sonarsource What do you guys think of the way of doing of radix? To basically attach all the sub-component to the parent component so that you can just import the parent component and then do stuff like this:
```ts
<DropdownMenu items={
  <>
    <DropdownMenu.GroupLabel/>
    <DropdownMenu.ItemButton/>
  </>
}>
```